tests/tensor_search/integ_tests/test_search_unstructured.py

```diff
@@ -689,7 +700,8 @@ def test_lexical_filtering(self):
             ("field1:(other things)", 1, "123456"),
             ("fake_int:234", 1, "other doc"),
             ("fake_float:1.23", 1, "other doc"),
-            ("gapped_field_name:gap", 1, "other doc")
+            ("gapped_field_name:gap", 1, "other doc"),
```

Comment:
qq we i don't see any field names with spaces - are we sure this works?

Reply:
It won't. We don't support spaces in field names